app/controllers/api/me_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::MeController < Api::BaseController
+
+  def update
+    user = User.find(current_user.id)
+    user.update!(me_params)
```

Comment:
Acá podés usar current_user de una:

```suggestion
    current_user.update!(me_params)
```